grakn-engine/src/main/java/ai/grakn/engine/controller/api/RelationshipController.java

```diff
@@ -123,6 +129,6 @@ private Json assignEntityAndRoleToRelationship(Request request, Response respons
 //    }
```

Comment:
Can kill this commented method

Reply:
@fppt  i missed those! good catch. fixed in ddc298e472a40a428129c4df7c7cfa6c75510c0a